packages/cli/src/templates/snow/hooks/write-trellis-context.py

```
@@ -0,0 +1,585 @@
+        if body:
+            lines.extend(["## .trellis/session/current.md", body, ""])
+    else:
+        sessions_dir = repo / ".trellis" / ".runtime" / "sessions"
+        if sessions_dir.is_dir():
+            try:
+                sessions = sorted(
+                    sessions_dir.glob("*.json"),
+                    key=lambda p: p.stat().st_mtime,
+                    reverse=True,
+                )
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)